### DISCUSSION TITLE
Disable rotation feature

### DISCUSSION THREAD

**Discussion by ghost** [lnreader/lnreader#342] [Ideas]
Hello, I usually have the rotation of my Android phone enabled.

But with hours of reading, I change my position of sitting or sometime laying down, and unintentionally, this will change the mode of reading from portrait mode to landscape mode.

When you go back to portrait mode the app won't get back exactly where it was the last time, making you scroll a bit.

If possible a button to disable rotation would be perfect.

**Comment by rajarsheechatterjee** [comment]
Create a [Feature Request](https://github.com/LNReader/lnreader/issues/new?assignees=&labels=Feature+Request&projects=&template=request_feature.yml) if there isn't an existing issue for it.
